test/runtests.jl

```diff
@@ -25,8 +25,8 @@ for test in [
         failed_tests += 1
         println(output)
         println(outbuf)
-        println(UTF8String(output))
-        println(UTF8String(outbuf))
+        println(String(output))
+        println(String(outbuf))
```

Comment:
Then it also needs to seed the `IOBuffer` and can't handle `output`. These cases are the reason for `Compat.UTF8String` is added and I don't see why it is not used....